pennylane/devices/qubit/adjoint_jacobian.py

```diff
@@ -108,3 +116,138 @@ def adjoint_jacobian(tape: QuantumTape):  # pylint: disable=too-many-statements
 
     # must be 2-dimensional
     return tuple(tuple(np.array(j_) for j_ in j) for j in jac)
+
+
+def adjoint_jvp(tape: QuantumTape, tangents: Tuple[Number]):
+    """Implements the adjoint method outlined in
+    `Jones and Gacon <https://arxiv.org/abs/2009.02823>`__ to differentiate an input tape.
+
+    After a forward pass, the circuit is reversed by iteratively applying adjoint
+    gates to scan backwards through the circuit.
+
+    .. note::
+        The adjoint differentiation method has the following restrictions:
```

Comment:
I don't think this is currently rendering anywhere but you will need a new line before this one if it ever is rendered somewhere

Reply:
also - do we validate the below things already in pre-processing?

Reply:
We already validate, but since this is a standalone function that can potentially be called elsewhere, it doesn't hurt to specify this explicitly in the docstring.